Packs/CommonScripts/Scripts/TextFromHTML/TextFromHTML.py

```diff
@@ -5,26 +5,57 @@
 import re
 
 
-def get_plain_text(html_regex):
+def get_plain_text(html: str):
     data = ''
-    if html_regex and html_regex.group(0):
-        data = re.sub(r'<.*?>', '', html_regex.group(0))
+    if html:
+        data = re.sub(r'<br>', '\n', html, flags=re.M + re.S + re.I + re.U)
+        data = re.sub(r'<.*?>', '', html, flags=re.M + re.S + re.I + re.U)
         entities = {'quot': '"', 'amp': '&', 'apos': "'", 'lt': '<', 'gt': '>', 'nbsp': ' ',
                     'copy': '(C)', 'reg': '(R)', 'tilde': '~', 'ldquo': '"', 'rdquo': '"', 'hellip': '...'}
         for e in entities:
             data = data.replace(f'&{e};', entities[e])
+        data = re.sub(r'[ \t]{2,}', ' ', data)
+        data = re.sub(r'(\r?\n){3,}', '\n\n', data)
+        data = re.sub(r'\s+$', '', data)
```

Comment:
These also change the current behavior of the script and break backwards-compatibility.
I'm also not sure if it's a good idea to "cleanup" the text, since some users might want to see it in the original format it was in untouched, just extracted from the HTML.

Reply:
This is now an Optional feature